File: src/main/java/de/cplaiz/activecraftcore/guis/offinvsee/OffInvSeeGui.java
package de.cplaiz.activecraftcore.guis.offinvsee;

import de.cplaiz.activecraftcore.guicreator.GuiCloseItem;
import de.cplaiz.activecraftcore.guicreator.GuiCreator;
import de.cplaiz.activecraftcore.guicreator.GuiCreatorDefaults;
import de.cplaiz.activecraftcore.guicreator.GuiItem;
import de.cplaiz.activecraftcore.messagesv2.MessageSupplier;
import de.cplaiz.activecraftcore.playermanagement.Profilev2;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.ToString;
import org.bukkit.entity.Player;

@Getter
@EqualsAndHashCode(callSuper = false)
@ToString
public class OffInvSeeGui extends GuiCreator {

    private final Player player;
    private final Player target;

    public OffInvSeeGui(Player player, Player target) {
        super("offinvsee", 3, "OffInv");
        this.player = player;
        this.target = target;
    }

    @Override
    public void refresh() {

        Profilev2 profile = Profilev2.of(player);
        setItem(new GuiCloseItem(GuiCreatorDefaults.closeItemDisplayname(profile.getMessageSupplier(GuiCreatorDefaults.acCoreMessage))), 22);
        fillBackground(true);

        setItem(new GuiItem(player.getInventory().getHelmet()).setMovable(true).setClickSound(null), 11);
        setItem(new GuiItem(player.getInventory().getChestplate()).setMovable(true).setClickSound(null), 12);
        setItem(new GuiItem(player.getInventory().getLeggings()).setMovable(true).setClickSound(null), 13);
        setItem(new GuiItem(player.getInventory().getBoots()).setMovable(true).setClickSound(null), 14);
        setItem(new GuiItem(player.getInventory().getItemInOffHand()).setMovable(true).setClickSound(null), 16);
    }
}

File: src/main/java/org/activecraft/activecraftcore/guis/offinvsee/OffInvSeeGui.kt
package org.activecraft.activecraftcore.guis.offinvsee

import org.activecraft.activecraftcore.guicreator.GuiCloseItem
import org.activecraft.activecraftcore.guicreator.GuiCreator
import org.activecraft.activecraftcore.guicreator.GuiCreatorDefaults
import org.activecraft.activecraftcore.guicreator.GuiCreatorDefaults.closeItemDisplayname
import org.activecraft.activecraftcore.guicreator.GuiItem
import org.activecraft.activecraftcore.playermanagement.Profile.Companion.of
import org.bukkit.entity.Player

class OffInvSeeGui(val player: Player, private val target: Player) : GuiCreator(identifier = "offinvsee", rows = 3, title = "OffInv") {
    override fun refresh() {
        val profile = of(player)
        setItem(GuiCloseItem(closeItemDisplayname(profile.getMessageSupplier(GuiCreatorDefaults.acCoreMessage!!))), 22)
        fillBackground(true)
        setItem(GuiItem(player.inventory.helmet!!).setMovable(true).setClickSound(null), 11)
        setItem(GuiItem(player.inventory.chestplate!!).setMovable(true).setClickSound(null), 12)
        setItem(GuiItem(player.inventory.leggings!!).setMovable(true).setClickSound(null), 13)
        setItem(GuiItem(player.inventory.boots!!).setMovable(true).setClickSound(null), 14)
        setItem(GuiItem(player.inventory.itemInOffHand).setMovable(true).setClickSound(null), 16)
    }
}